fffw/graph/base.py

```diff
@@ -59,7 +59,14 @@ def name(self) -> str:
 
     @property
     def meta(self) -> Optional[Meta]:
-        return self.get_meta_data(self)
+        metadata = self.get_meta_data(self)
+        if metadata is None:
+            return None
+        return self.transform(metadata)
+
+    def transform(self, *metadata: Meta) -> Meta:
```

Comment:
Codec accepts only single input stream. There is no need to support multiple, if there are no complains from MyPy.

Reply:
Yes, MyPy found conflict:
```
Definition of "transform" in base class "Dest" is incompatible with definition in base class "Node"
```
I'm not sure why but Filter definition inherits both Node and Dest for TYPE_CHECKING:
```
class Filter(mixins.StreamValidationMixin, base.Node, Params)
...
class StreamValidationMixin(StreamValidationTarget)
...
if TYPE_CHECKING:
    StreamValidationTarget = base.Dest
else:
    StreamValidationTarget = object
```

Reply:
Both Node and Dest have input edges, it's the main reason.